<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExchangeInfo</class>
 <widget class="QWidget" name="ExchangeInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>546</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Exchange Info</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_filter"/>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_updatePrice">
     <property name="text">
      <string>Update Price</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_isTrading">
       <property name="text">
        <string>Hide UnTrade</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_quotaAsset">
       <item>
        <property name="text">
         <string>USDT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BUSD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BTC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ALL</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_updateInfo">
     <property name="text">
      <string>Update Info</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_addALL">
     <property name="text">
      <string>Add All</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
